root/types.js

```diff
@@ -610,6 +610,16 @@ declare type MaybeGroupedOptionsT =
   | {|+grouped: true, +options: GroupedOptionsT|}
   | {|+grouped: false, +options: SelectOptionsT|};
 
+declare type MediumFormatT = {|
```

Comment:
I think a `MediumFormatT` definition has been added since this PR was opened, so you might want to merge them or remove this one.